src/pam-ssh-add/Makefile.am

```diff
@@ -1,17 +1,19 @@
 noinst_LIBRARIES += libpam_ssh_add.a
 
 libpam_ssh_add_a_SOURCES = \
+	src/common/cockpitcloserange.h \
+	src/common/cockpitcloserange.c \
 	src/pam-ssh-add/pam-ssh-add.c \
 	src/pam-ssh-add/pam-ssh-add.h \
 	$(NULL)
 
-libpam_ssh_add_a_CFLAGS = $(NULL)
+libpam_ssh_add_a_CFLAGS = -fPIC
```

Comment:
...and what's this?

Reply:
It was a bug in the Makefile which didn't surface until trying to add a second .c file. Fixed in a separate commit.